rpc/client.go

```diff
@@ -198,6 +198,13 @@ func getSignedURL(opts *UploadReportOptions) (string, error) {
 	}
 
 	defer response.Body.Close()
+	grip.Debug(message.Fields{
+		"Status Code": response.StatusCode,
+	})
+	if response.StatusCode != http.StatusOK {
+		return "", errors.Errorf("Non-OK HTTP status. Status Code: %d. Status Text: %s", response.StatusCode, http.StatusText(response.StatusCode))
```

Comment:
Can you change this to `"failed to get the Data-Pipes signed URL with response '%s'", response.Status`.